lib/kredis/types/hash.rb

```diff
@@ -1,28 +1,37 @@
 require "active_support/core_ext/hash"
 
 class Kredis::Types::Hash < Kredis::Types::Proxying
-  proxying :hget, :hset, :hmget, :hdel, :hgetall, :hkeys, :hvals, :del, :exists?
+  ZERO_FIELDS_ADDED = 0
+
+  proxying :hset, :hdel, :hgetall, :del, :exists?, :multi, :callnx
```

Comment:
I removed some of these proxying methods because they were no longer used in the class.  Do they need to be kept because of the public api others may have tied into?